src/proto/FunctionRpc.proto

```diff
@@ -0,0 +1,256 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+option java_multiple_files = true;
+option java_package = "com.microsoft.azure.webjobs.script.rpc.messages";
+option java_outer_classname = "FunctionProto";
+option objc_class_prefix = "FunctionRpc";
+option csharp_namespace = "Microsoft.Azure.WebJobs.Script.Grpc.Messages";
+
+package FunctionRpc;
+
+import "google/protobuf/duration.proto";
+
+// Interface exported by the server.
+service FunctionRpc {
+ rpc EventStream (stream StreamingMessage) returns (stream StreamingMessage) {}
+}
+
+message StreamingMessage {
+  string request_id = 1;
+  oneof content {
+    StartStream start_stream = 20; 
+
+    // Host sends capabilities/init data to worker
+    WorkerInitRequest worker_init_request = 17;
+    // Worker responds after initializing with its capabilities & status
+    WorkerInitResponse worker_init_response = 16;
+
+    // Worker periodically sends empty heartbeat message to host
+    WorkerHeartbeat worker_heartbeat = 15;
+
+    // Host sends terminate message to worker.
+    // Worker terminates if it can, otherwise host terminates after a grace period
+    WorkerTerminate worker_terminate = 14;
+
+    // Add any worker relevant status to response
+    WorkerStatusRequest worker_status_request = 12;
+    WorkerStatusResponse worker_status_response = 13;
+
+    // On file change event, host sends notification to worker
+    FileChangeEventRequest file_change_event_request = 6;
+
+    // Worker requests a desired action (restart worker, reload function)
+    WorkerActionResponse worker_action_response = 7;
+    
+    // Host sends required metadata to worker to load function
+    FunctionLoadRequest function_load_request = 8;
+    // Worker responds after loading with the load result
+    FunctionLoadResponse function_load_response = 9;
+    
+    InvocationRequest invocation_request = 4;
```

Comment:
Can you please add missing comments describing message types? [VS Code extesion](https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3) uses this to show documentation